manifest.json

```diff
@@ -1,17 +1,30 @@
 {
     "spark_version": "2.4.0",
     "default_spark_dist": {
+        "scala_version": "2.11",
         "hadoop_version": "2.9",
-        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.4.0-bin-hadoop2.9.2-openjdk8-SNAPSHOT-20190530.tgz"
+        "uri": "https://downloads.mesosphere.io/spark/assets/spark-2.4.3-bin-scala-2.11-hadoop-2.9.tgz"
```

Comment:
A few comments on this:
- let's please use `mesosphere.com` instead of `.io`
- let's keep asset naming consistent, e.g.: `spark-2.4.3-bin-hadoop-2.9-scala-2.11-SNAPSHOT-20190822.tgz` (timestamp is important to have an idea of when this build has been created)
- let's avoid using prod assets bucket for non-release distributions and store artifacts in `infinity-artifacts` bucket

Reply:
Updated the naming and moved to `infinity-artifacts` bucket